VARIABLES.md

```diff
@@ -69,3 +70,31 @@
 | secondary_vpcs | List of VPCs to associate with Service Discovery namespace. | list | `<list>` | no |
 | table_param_filter | A regular expression for selecting necessary table parameters for the SNS listener. If the value isn't set, then no table parameters are selected. | string | `` | no |
 | vpc_id | VPC ID. | string | - | yes |
+
+### apiary_assume_roles
+
+A list of maps.  Each map entry describes a role that is created in this account, and a list of principals (IAM ARNs) in other accounts that are allowed  
+to assume this role.  Each entry also specifies a list of Apiary schemas that this role is allowed to write to.
+
+An example entry looks like:
+```
+apiary_assume_roles = [
+  {
+    name = "client_name"
+    principals = [ "arn:aws:iam::account_number:role/cross-account-role" ]
+    schema_names = [ "dm","lz","test_1" ]
+    max_session_duration = "7200",
```

Comment:
Can we rename this to `max_role_session_duration_seconds` to match what I've seen AWS call it?

Reply:
will do